multiplier/Cargo.toml

```diff
@@ -1,35 +0,0 @@
-[package]
```

Comment:
This is a useful example for onboarding and understanding roughly how things fit together. I don't think it makes sense to remove this package.

We could move it to an example directory.